DataFormats/HGCalReco/src/classes_def.xml

```diff
@@ -1,19 +1,16 @@
 
 <lcgdict>
-  <class name="ticl::Trackster" ClassVersion="5">
-   <version ClassVersion="5" checksum="3878166595"/>
+  <class name="ticl::Trackster">
```

Comment:
Just to be sure, if the Trackster class is planned to be persisted in production, then the class version is still useful.
From earlier comments I understood that it may be used for storage mostly for tests or otherwise cases where the format evolution would not matter.
@rovere 
please clarify/confirm.
Thank you.

Reply:
@slava77 that's difficult to say at the moment. The idea is to use them as intermediate products to deliver something more complex and with more links with the outside world.
I'll add it back and, in case we will end up never persisting it, we can remove it at a later time. Would that be ok?

Reply:
>  I'll add it back and, in case we will end up never persisting it, we can remove it at a later time. Would that be ok?

Yes, please.